hello.html

```diff
@@ -0,0 +1 @@
+hello
```

Comment:
```suggestion
ハロー
```
ここは日本語対応でハローでは無いでしょうか?